platform.md

```diff
@@ -1,10 +1,9 @@
 # Platform Interface Specification
 
-This document specifies the interface between a lifecycle and a platform.
+This document specifies how to process buildpack lifecycle steps to ensure compatibility
+between end-user tools.
 
-A platform orchestrates a lifecycle to make buildpack functionality available to end-users such as application developers.
```

Comment:
Could we keep this line?

Reply:
Explaining what a platform does instead of what platform is is exactly what makes this doc so confusing. There is an interface. The interface should be between something and something. Usually the interface is API, or CLI, or a protocol. When this context is clear, it could be said that `platform` is an command line app or server backend implemeting the API. But a straight mention that "some platform orchestrates some lifecyce" is like missing some links in the logic chain.